### PR TITLE
Update project dependencies.

### PR DESCRIPTION
**Pull request by eiloncap**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,19 +1,19 @@
 buildscript {
-    ext.kotlin_version = '1.3.72'
+    ext.kotlin_version = '1.7.0'
```
Comment (bmax-ltx) [inline build.gradle]: For consistency let's use the same versions of Kotlin and Gradle that we have in `facetune-android`:
`ext.kotlin_version = "1.5.30"`
`classpath 'com.android.tools.build:gradle:7.0.4'`

File: app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt [hunk 2]
```diff
@@ -36,7 +36,7 @@ open class FeedViewModel : ViewModel() {
  * dependencies into [FeedViewModel] in the future, then this is the place to do it.
  */
 class FeedViewModelFactory : ViewModelProvider.Factory {
```
Comment (bmax-ltx) [inline app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt]: I think we should also change to it to:
`class FeedViewModelFactory(private val context: Context) : ViewModelProvider.NewInstanceFactory() {`

**Review by bmax-ltx — COMMENTED** [review]
Thank you!

**Comment by eiloncap** [comment]
Resolved

**Review by bmax-ltx — APPROVED** [review]
Cool!

**Pull request merged**